### PR TITLE
i18n: reverse direction of some gridicons in RTL

### LINKED ISSUE #1734: Editor: back button is backwards in RTL ui
screen shot 2015-10-21 at 3 32 01 pm

We want to go back, not forward.

### PR DESCRIPTION
**Pull request by yoavf**
Across the codebase, we're often using specific gridicons to emphasis directionality: for example, `chevron-right` to expand a compact card, or `arrow-left` for back links. In RTL, those icons needs to be reversed.

The best semantical solution would be to abstract the arrows into a direction aware component, and then make sure we use this component everywhere. However, I feel this would add too much complexity, and propose a workaround we've used before: reverse using css.

This fixes #1734, and also other various issues (see screenshots below) 

**Before patch**

Issue in domains list compact card: 
screen shot 2016-02-02 at 10 06 50

Issue in domains back link: 
screen shot 2016-02-02 at 10 39 05

Issue in editor back link:
screen shot 2015-10-21 at 3 32 01 pm

**After patch:**
domains list compact card: 
screen shot 2016-02-02 at 10 06 29

domains back link: 
screen shot 2016-02-02 at 10 41 40

editor back link:
screen shot 2016-02-02 at 10 40 52

### CODE REVIEW

File: assets/stylesheets/layout/_main.scss [hunk 1]
```diff
@@ -100,6 +100,21 @@ body.rtl,
 	font-family: $sans;
 }
 
+/*rtl:ignore*/
+.rtl {
+	svg {
```
Comment (mtias) [inline assets/stylesheets/layout/_main.scss]: Do we need the svg selector here?

**Comment by mtias** [comment]
This is a good solution. Do we have to identify those that may already be doing the reverse but by a different method? For instance:

```
.rtl .current-site__switch-sites .gridicon {
	transform: rotate( 180deg );
}
```

File: assets/stylesheets/layout/_main.scss (hunk 1 above)
Comment (yoavf) [inline assets/stylesheets/layout/_main.scss]: Nope, I'll fix - thanks

**Comment by yoavf** [comment]
@mtias 
> Do we have to identify those that may already be doing the reverse but by a different method? 
Yes - I wasn't able to find any other reversals of gridicons so I think that should do it. Thanks!

**Comment by mtias** [comment]
Perfect!

**Comment by mtias** [comment]
:+1:

**Pull request merged**